controllers/story-controller.ts

```diff
@@ -58,12 +59,13 @@ export default class StoryController implements StoryControllerI {
   createStory = (req: Request, res: Response) => {
     // @ts-ignore
     const userUid = req.params.uid === "me" && req.session['profile'] ? req.session['profile']._id : req.params.uid;
-      if(userUid === "me"){
+    const viewers = req.body.viewers;
+      if(userUid === "me" || viewers.includes(userUid)){
         res.sendStatus(503);
         return;
       }
       try {
-        StoryController.storyDao.createStory(userUid, req.body).then((story: Story) => res.json(story));
+        StoryController.storyDao.createStory(userUid, req.body.viewers, req.body).then((story: Story) => res.json(story));
```

Comment:
`viewers` are already present in `req.body` right? Why do we need to pass the as a separate method parameter?

Reply:
I have deleted passing a viewer as a separate parameter everywhere. Getting it from body.